_data/concepts.yml

```diff
@@ -20,6 +20,7 @@ toc:
   section:
   - docs/concepts/architecture/nodes.md
   - docs/concepts/architecture/master-node-communication.md
+  - docs/concepts/architecture/apiserver-aggregation.md
```

Comment:
Ok, I'm going to create a new subsection under Concepts called *Extending the Kubernetes API* and put this in there. We can then start adding other docs into that. I did originally try to put it in the Overview subsection next to *Kubernetes API*, but this doc isn't really an overview so it didn't quite fit.